app/services/unified_chat_service.py

```
@@ -0,0 +1,1261 @@
+            # For now, returning structure that matches your system
+            return {
+                "trading_mode": "balanced",
+                "operation_mode": "assisted",
+                "risk_tolerance": "medium",
+                "notification_preferences": {}
+            }
+        except Exception as e:
+            self.logger.error("Failed to get user config", error=str(e))
+            return {"trading_mode": "balanced", "operation_mode": "assisted"}
+    
+    async def _get_performance_metrics(self, user_id: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused method argument: `user_id`

(ARG002)